config/environments/development.rb

```diff
@@ -55,4 +55,5 @@
     Bullet.bullet_logger = true
     Bullet.rails_logger = true
   end
+  config.factory_bot.definition_file_paths = ["spec/factories"]
```

Comment:
Why do you need this definition? :thinking:

Reply:
because if i didn't put that it dont recognize me the factories that i'm using

Reply:
:thinking: I have run the test suite without that line and it works just fine. Also in the terminal it creates the elements just fine:

```
irb(main):001:0> FactoryBot.create(:contributor)
   (4.4ms)  BEGIN
  Contributor Create (1.1ms)  INSERT INTO "contributors" ("name", "workload", "admission_date", "created_at", "updated_at") VALUES ($1, $2, $3, $4, $5) RETURNING "id"  [["name", "vero"], ["workload", 0], ["admission_date", "2020-02-14"], ["created_at", "2020-02-21 16:18:34.922587"], ["updated_at", "2020-02-21 16:18:34.922587"]]
   (0.7ms)  COMMIT
=> #<Contributor id: 1, name: "vero", workload: "20", country: "argentina", admission_date: "2020-02-14", contributor_type: "contributor", created_at: "2020-02-21 16:18:34", updated_at: "2020-02-21 16:18:34">
```